rqd/rqd/rqlogging.py

```diff
@@ -0,0 +1,149 @@
+#  Copyright Contributors to the OpenCue Project
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+"""Logging module, handles logging to files and non-files"""
+
+
+import logging
+import re
+import time
+import os
+import datetime
+import platform
+
+import rqd.rqconstants
+
+LOGTYPE_FILE = 1
+
+log = logging.getLogger(__name__)
+log.setLevel(rqd.rqconstants.CONSOLE_LOG_LEVEL)
+
+
+class RQDLogger(object):
```

Comment:
Rename class to RqdLogger, this is the [preferred way to handle acronyms on camelCase](https://en.wikipedia.org/wiki/Camel_case#Programming_and_coding)

Reply:
Done